src/components/StudyList/StudyTable.js

```diff
@@ -61,27 +64,33 @@ const StudyTable = ({
   columns,
 }) => {
   const [sorting, setSorting] = useState({
-    column: 'name',
-    direction: 'descending',
+    column:
```

Comment:
Could we move this up into the `StudyList` component along with the rest of the column state:
https://github.com/kids-first/kf-ui-data-tracker/blob/master/src/components/StudyList/StudyList.js#L68-L99
That will make the one column state variable the single key in localStorage that deals with all the table display values.